### ISSUE TITLE
NVDA add-on should be updated for NVDA 2022.1.

### ISSUE THREAD

**Issue by jasonjgw** [NSoiffer/MathCATForPython#13]
The current add-on is incompatible with the beta release of NVDA 2022.1. Backward-incompatible API changes are documented in [these release notes](https://www.nvaccess.org/files/nvda/releases/2022.1beta1/documentation/changes.html).

**Comment by NSoiffer** [comment]
I updated the manifest and a new (untested) version is at https://github.com/NSoiffer/MathCATForPython/releases/download/v0.1.7/MathCAT-0.1.7-2022.1.beta.nvda-addon

@jasonjgw: since you have the beta (and I want to make sure MathCAT still works with the release), could you give new version a try?

**Comment by jasonjgw** [comment]
I tested it successfully with NVDA 2022.1 beta 2. I think you can safely push the manifest change to the Git repository and close the issue.
